vendor/engines/secure_rooms/spec/controllers/secure_rooms_api/scans_controller_spec.rb

```diff
@@ -31,7 +31,7 @@
           let(:card_user) { build :user }
 
           it { is_expected.to have_http_status(:not_found) }
-          it "is expected to contain the corresponding reason" do
+          xit "is expected to contain the corresponding reason" do
```

Comment:
these are pending in this commit because there aren't yet any response objects with errors